docs/configuration.md

```diff
@@ -685,23 +685,23 @@ Apart from these, the following properties are also available, and may be useful
   <td><code>spark.ui.retainedJobs</code></td>
   <td>1000</td>
   <td>
-    How many jobs the Spark UI and status APIs remember before garbage collecting. 
+    How many jobs the Spark UI and status APIs remember before garbage collecting.
```

Comment:
Yea, my editor does this automatically.  I'll revert it.  It would be nice if the style checker enforces this, though.